verl/models/registry.py

```diff
@@ -17,6 +17,32 @@
 
 import torch.nn as nn
 
+# Supported models using HF Rmpad
+# TODO(sgm): HF may supported more than listed here, we should add more after testing
+from transformers import LlamaConfig, MistralConfig, GemmaConfig, Qwen2Config
+
+_RMPAD_MODELS = {
```

Comment:
This may cause issue for reward model and value model because they are xxxForTokenClassification?

Reply:
Shall we add a e2e CI that guarantees runnable? Just use QWen2.5-0.5b to run one step

Reply:
Make sure all the features are toggled

Reply:
> This may cause issue for reward model and value model because they are xxxForTokenClassification?

This would be fine as the `config.architectures` field from the AutoConfig is always xxxForCausal.
But it's true that using architectures is not that straightforward. I'll use model_type instead

Reply:
> Shall we add a e2e CI that guarantees runnable? Just use QWen2.5-0.5b to run one step

The ray_trainer may not have an option to run one step. One epoch is feasible.